### DISCUSSION TITLE
How to uninstall?

### DISCUSSION THREAD

**Discussion by chakko17** [LostArtefacts/TRX#735] [Q&A] [answered]
Hey,

maybe a stupid question, but, how do you uninstall Tomb1Main? Just delete the files from the directory you installed to? No leftovers on the system, in AppData, Documents, or the registry?

**Comment by walkawayy** [comment, marked as answer]
Yep just delete the folder to uninstall @chakko17. The game doesn't touch the registry or other folders.

**Reply by chakko17** [reply]
Thank you. 👍
